src/ingestion/sources/bigquery.py

```diff
@@ -0,0 +1,200 @@
+import json
+
+from ingestion.task import TaskUnit
+
+from .base import MetricsQueryBuilder, SourceConfiguration, SQLAlchemySourceDialect, SQLAlchemySource
+from sqlalchemy.engine.url import make_url
+
+class BigQueryMetricsQueryBuilder(MetricsQueryBuilder):
+    def _base_query_sample(self, select_sql):
+        return """
+            SELECT 
+                CURRENT_TIMESTAMP as `WINDOW_START`, 
+                CURRENT_TIMESTAMP as `WINDOW_END`, 
+                COUNT(*) as `ROW_COUNT`, 
+                '{table}' as `TABLE_NAME`,
+                '{database}' as `DATABASE_NAME`,
+                '{schema}' as `SCHEMA_NAME`,
+
+                {select_sql}
+            FROM {schema}.{table};
```

Comment:
Does this need database identifier?

Is there something else about this query that is different from the base?

Reply:
Not necessary but will add for consistency. Yes, the way BigQuery takes string literal column names is with back ticks rather than quotes, thus those change as well